boot/libs.ml

```diff
@@ -1,4 +1,4 @@
-let external_libraries = [ "unix"; "threads" ]
+let external_libraries = [ "threads.posix" ]
```

Comment:
The CI failure is probably because of this change.

Reply:
Our script is:
```
make release
_build/default/bin/dune.exe build @fmt --auto-promote
```
Which promotes this change. If I run this again locally, the change is undone. Do you have an idea on what's happening ?

Reply:
There's some trickery around `threads.posix` in `src/dune_rules/bootstrap_info.ml` but I don't know exactly what is happening there and why the inputs seem to be different. Looking at the mention of `threads.posix` in the odoc rules it seems like `threads` is somewhat arcanely set up and kinda aliases to `threads.vm` or `threads.posix`.

Though I think it it would be better to run `make bootstrap` followed by `./dune.exe fmt` to apply the formatting.

Reply:
That's fixed now.